modules/ROOT/pages/introduction-to-tiny-spellchecker.adoc

```diff
@@ -42,7 +41,6 @@ tinymce.init({
 
 To enable the {productname} Enterprise Spellchecking plugin:
 
-. If configured, disable the `+spellchecker+` plugin provided with {productname}, by removing it from the `+plugins+` list.
 . Add `+tinymcespellchecker+` to the `+plugins+` list.
```

Comment:
Tyler was against lists with a single item and I'd agree we don't want that in this case. So should this just collapse into a single sentence now? e.g
```
To enable the {productname} Enterprise Spellchecking plugin, add `+tinymcespellchecker+` to the `+plugins+` list.
```

- [ ] Consider collapsing this into a sentence instead of a list

Reply:
Yeah, I wasn't sure about that. Have collapsed it in both places now. Thanks